code/src/components/products-grid/ProductsGrid.tsx

```diff
@@ -0,0 +1,31 @@
+import AppBox from "@/components/app-box/AppBox";
+import ProductCard from "@/components/product-card/ProductCard";
+import ProductSkeleton from "@/components/product-skeleton/ProductSkeleton";
+import { ProductsGridProps } from "@/components/products-grid/ProductsGrid.types";
+
+import { Product } from "@/types/product.types";
+import cn from "@/utils/cn/cn";
+import repeatComponent from "@/utils/repeat-component/repeatComponent";
+
+import "@/components/products-grid/ProductsGrid.scss";
+
+const ProductsGrid = ({
+  products,
+  className,
+  isLoading = false,
+  loadingItemsCount = 5
+}: ProductsGridProps) => {
+  const skeletonCards = repeatComponent(<ProductSkeleton />, loadingItemsCount);
+
+  const productCards = products?.map((product: Product) => (
+    <ProductCard key={product.id} product={product} />
+  ));
+
+  const gridItems = isLoading ? skeletonCards : productCards;
```

Comment:
Can we also handle error case here? We can render some error fallback instead of empty value

Reply:
I guess it can be added

Reply:
Added ability to pass `isError` property and simple fallback message